src/main/java/com/compilercharisma/chameleonbusinessstudio/controller/AppointmentController.java

```diff
@@ -31,18 +31,18 @@ public class AppointmentController {
 
     private final AppointmentService appointments;
     private final AuthenticationService authentication;
-//    private final PagedResourcesAssembler<AppointmentEntity> asm;
-    private final AppointmentModelAssembler modelAssembler;
+    //private final PagedResourcesAssembler<AppointmentEntity> asm;
```

Comment:
Should we just remove the code completely instead of commenting?

Reply:
I'm commenting it out so we know what we need to replace. I know a lot of the dependencies we've had to replace are usually just a matter of replacing `servlet` in the fully qualified class name with `reactive`. Basically, this prevents us from forgetting what terminology Spring uses for `thingThatConvertsListToPage`